tsconfig.json

```
@@ -2,17 +2,25 @@
   "compilerOptions": {
     "target": "esnext",
     "module": "esnext",
-    "lib": ["es2015", "dom"],  // Add this line to enable ES2015 and DOM types
+    "lib": [
+      "es2015",
+      "es2016",
+      "dom"
+    ], // Add this line to enable ES2015 and DOM types
```

It's not a line anymore, so I think we no longer need the comment.
```suggestion
    ],
```